pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -208,13 +209,6 @@ func (pl *VolumeBinding) PreBind(ctx context.Context, cs *framework.CycleState,
 // Unreserve clears pod binding state.
 // TODO(#90962) Revert assumed PV/PVC cache
 func (pl *VolumeBinding) Unreserve(ctx context.Context, cs *framework.CycleState, pod *v1.Pod, nodeName string) {
```

Comment:
after https://github.com/kubernetes/kubernetes/pull/91705, we need to revert assumed PV/PVC cache in this extension point